datalogistik/datalogistik.py

```diff
@@ -60,7 +60,11 @@ def main():
         log.debug(
             f"Found cached dataset metadata file at '{cached_dataset_metadata_file}'"
         )
-        util.copy_from_cache(cached_dataset_path, argument_info.dataset)
+        if argument_info.output:
+            output_dir = pathlib.Path(argument_info.output, argument_info.dataset)
```

Comment:
I wonder if we should just take the given output dir literally instead of creating a subfolder with the dataset name?
```suggestion
            output_dir = pathlib.Path(argument_info.output)
```

Reply:
oh if you think that's a good idea you'll have to change it down below too

Reply:
Now no longer creating a subfolder with the dataset name.